napari/_qt/qt_application.py

```diff
@@ -0,0 +1,23 @@
+from qtpy.QtCore import Slot
+from qtpy.QtWidgets import QApplication
+
+from .dialogs.qt_notification import NapariQtNotification
+
+
+class NapariQApplication(QApplication):
```

Comment:
I'm good with this, but maybe @tlambert03 can just weigh in if he likes this pattern as well

Reply:
No longer needed.